http/errors.go

```diff
@@ -23,6 +23,7 @@ const (
 	errMethodIsNotImplemented       string = "the method is not implemented"
 	errFailedToGetContext           string = "failed to get context"
 	errPurgeRequestNonDeveloperMode string = "cannot purge database when development mode is disabled"
+	errCanNotPurgeOnlyACPState      string = "cannot purge only ACP state, use high-level DB purge instead"
```

Comment:
todo: The new error can be removed now :)

Reply:
- [x] Done